src/example-types/facet.js

```diff
@@ -38,6 +61,13 @@ module.exports = {
       ]),
     ]).then(([options, cardinality]) => ({
       cardinality: _.get('0.count', cardinality),
-      options: _.map(x => ({ name: x._id, count: x.count }), options),
+      options: _.map(
+        x => ({
```

Comment:
It might be cleaner if you destructure x to `{ _id, labelData, count }` here. Then all the objects bellow would be simpler.

Reply:
yeah, good point, I'll clean that up